lib/reversetunnel/localsite.go

```diff
@@ -153,6 +153,10 @@ func (s *localSite) DialAuthServer() (conn net.Conn, err error) {
 		return nil, trace.Wrap(err)
 	}
 
+	if len(authServers) < 1 {
+		return nil, trace.ConnectionProblem(nil, "no auth servers available")
```

Comment:
Same here `no auth nodes available`

Reply:
From what I can tell, it appears that we typically refer to the `ssh_service` as a node, but always use `server` when referring to the `auth_service` (e.g. `tctl --auth-server=example.com`).

Reply:
👍 Yeah. I see what you mean.. I was just running `tctl status --auth-server=`